pages/space-between/permutations.page.tsx

```diff
@@ -8,35 +8,55 @@ import ScreenshotArea from '../utils/screenshot-area';
 import styles from './styles.scss';
 import Container from '~components/container';
 
-const ExampleContent = () => <div className={styles.box}>This is some text.</div>;
+const ExampleContent = ({ renderNull }: { renderNull?: boolean }) =>
+  renderNull ? null : <div className={styles.box}>This is some text.</div>;
 
 /* eslint-disable react/jsx-key */
 const permutations = createPermutations<SpaceBetweenProps & { note?: string }>([
   {
     size: ['xxxs', 'xs', 's', 'm', 'l', 'xl', 'xxl'],
     direction: ['vertical', 'horizontal'],
-    children: [[<ExampleContent />, <ExampleContent />, <ExampleContent />]],
+    children: [
+      [
+        <ExampleContent renderNull={true} />,
+        <ExampleContent />,
+        <ExampleContent renderNull={true} />,
+        <ExampleContent />,
+        <ExampleContent />,
+        <ExampleContent renderNull={true} />,
+      ],
+    ],
   },
   {
     size: ['xs', 'xl'],
     direction: ['horizontal', 'vertical'],
     children: [
       [
+        () => null,
```

Comment:
This element gets filtered out by `react-keyed-flatten-children` library used in space between component (same will happen even if it returns a non-empty value) 

Is this the behavior you wanted to test here?

Reply:
No, good callout, will adjust